docs/src/manual.md

```diff
@@ -857,10 +857,23 @@ A common workflow used in BenchmarkTools is the following:
 
 1. Start a Julia session
 2. Execute a benchmark suite using an old version of your package
-3. Save the results somehow (e.g. in a JSON file)
+    ```julia
+    results = run(suite, verbose = true)
+    ```
+4. Save the results somehow (e.g. in a JSON file)
+    ```julia
+    BenchmarkTools.save("results.json", median(results))
+    ```
 4. Start a new Julia session
 5. Execute a benchmark suite using a new version of your package
-6. Compare the new results with the results saved in step 3 to determine regression status
+   ```julia
+   results = run(suite, verbose = true)
+   ```
+7. Compare the new results with the results saved in step 3 to determine regression status
+    ```julia
+    old_results = BenchmarkTools.load("results.json")
+    BenchmarkTools.judge(median(results), old_results)
```

Comment:
For consistency, it would be nice if `results` and `old_results` were comparable (either both aggregated or none of them)

Reply:
going for the minimum since that is the default quantity in BenchmarkTools

Reply:
tweaking a little